### PR TITLE
Connect accounts

### PR DESCRIPTION
**Pull request by Andrewangeta**

### CODE REVIEW

File: Sources/Models/Connect/Account.swift [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    
+    public init(node: Node) throws {
+        self.id = try node.get("id")
+        self.object = try node.get("object")
+        self.businessName = try node.get("business_name")
+        self.businessUrl = try node.get("business_url")
+        self.chargesEnabled = try node.get("charges_enabled")
+        // TODO - investigate, not getting value from response after rejecting account. 
+        // https://stripe.com/docs/api/curl#reject_account
+        // self.transfersEnabled = try node.get("transfers_enabled")
```
Comment (anthonycastelli) [inline Sources/Models/Connect/Account.swift:57]: Do we need to keep these comments in here?

File: Sources/Models/Connect/Account.swift [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    }
+    
+    public func makeNode(in context: Context?) throws -> Node {
+        let object: [String: Any?] = [
+            "id": self.id,
+            "object": self.object,
+            "business_name": self.businessName,
+            "business_url": self.businessUrl,
+            "charges_enabled": self.chargesEnabled,
+            //"transfers_enabled": self.transfersEnabled,
```
Comment (anthonycastelli) [inline Sources/Models/Connect/Account.swift:91]: Same thing here

**Review by anthonycastelli — CHANGES_REQUESTED** [review]

File: Sources/Models/Connect/Account.swift (hunk 1 above)
Comment (Andrewangeta) [reply Sources/Models/Connect/Account.swift:57]: Not necessarily If you're aware of them it can be deleted. but it was a weird issue as the comment says.

Comment (anthonycastelli) [reply Sources/Models/Connect/Account.swift:57]: Is it an issue on Stripe's side or our end?

Comment (Andrewangeta) [reply Sources/Models/Connect/Account.swift:57]: More of stripe. It's confusing. It studs that I should get this return value when rejecting an account but I never see it.

Comment (anthonycastelli) [reply Sources/Models/Connect/Account.swift:57]: Oh okay, I gotcha. Then, lets leave it in there

**Review by anthonycastelli — APPROVED** [review]

**Pull request merged**